ember-headlessui/addon/components/combobox.js

```diff
@@ -21,15 +20,14 @@ const PREVENTED_KEYDOWN_EVENTS = new Set([
 ]);
 
 export default class ComboboxComponent extends Component {
-  @tracked selectedOptionGuids = new TrackedSet();
   @tracked _isOpen = this.args.isOpen || false;
   //TODO: `options` would be a better name as we store other stuff in it, too.
   @tracked optionElements = [];
   @tracked activateBehaviour = ACTIVATE_NONE;
   @tracked _activeOptionGuid;
+  options = [];
```

Comment:
Also thinking of the `TODO` above, do we need both `optionElements` and `options`?

Reply:
As I described above, eventually I would like to remove modifier callback entirely (`this.registerOptionElement`) and `optionElements` collection as well. This PR is a first step.
Answering our question, if we can merge them (`registerOption` and `registerOptionElement`) — the ideal solution would be to remove register modifier, and only keep constructor / destructor if necessary (for select / active element) but not DOM element what we currently do.